From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 13 Aug 2019 14:53:13 +0100
Subject: Revert "xen/pciback: Don't disable PCI_COMMAND on PCI device reset."
Forwarded: not-needed

This fix is incompatible with the version of qemu in jessie.

---
--- a/drivers/xen/xen-pciback/pciback_ops.c
+++ b/drivers/xen/xen-pciback/pciback_ops.c
@@ -126,6 +126,8 @@ void xen_pcibk_reset_device(struct pci_d
 		if (pci_is_enabled(dev))
 			pci_disable_device(dev);
 
+		pci_write_config_word(dev, PCI_COMMAND, 0);
+
 		dev->is_busmaster = 0;
 	} else {
 		pci_read_config_word(dev, PCI_COMMAND, &cmd);
